frontend/src/v5/ui/routes/viewer/viewer.tsx

```diff
@@ -80,6 +82,11 @@ export const Viewer = () => {
 		ViewerActionsDispatchers.fetchData(teamspace, project, containerOrFederation);
 	}, [teamspace, project, containerOrFederation]);
 
+	useEffect(() => {
```

Comment:
I dont understand why you have this bug in this branch but not in staging.

Reply:
This was fixed in #4906, but as the code makes sense to be here as well, I'm going to add it to this as well